internal/lightcone/nihility/eyesoftheprey/eyesoftheprey.go

```diff
@@ -0,0 +1,37 @@
+package eyesoftheprey
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	mod key.Modifier = "eyes_of_the_prey"
+)
+
+// Increases the wearer's Effect Hit Rate by 20%/25%/30%/35%/40% and increases DoT by 24%/30%/36%/42%/48%.
+func init() {
+	lightcone.Register(key.EyesofthePrey, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_NIHILITY,
+		Promotions:    promotions,
+	})
+}
+
+func Create(engine engine.Engine, owner key.TargetID, lc info.LightCone) {
+	ehr_amt := 0.2 + 0.05*float64(lc.Imposition)
+	dot_amt := 0.24 + 0.06*float64(lc.Imposition)
+
+	engine.AddModifier(owner, info.Modifier{
+		Name: EyesofthePrey,
+		Source: owner,
+		Stats: info.PropMap{prop.EffectHitRate: ehr_amt, prop.DOTDamagePercent: dot_amt},
+		State: Amts{ehr: ehr_amt, dot: dot_amt},
+	})
```

Comment:
* `State` is only used when you need to pass data from creation to listener logic (like if this modifier does something `OnBeforeHit` or something). Since you are just setting the stats on add, `State` does nothing here
* Should use the `mod` name you declared as a const. With current setup, you are using the lightcone name as the modifier name (though functionally no difference)
* Typically you want to register all the modifiers created; however, in this case registration is not strictly necessary since you do not need anything beyond the blank/default config.